minimal responsive with pure.css

Kashouryo opened the pull request:
Basic responsive implementation for better view on mobile devices.

Before:
![image](https://user-images.githubusercontent.com/37679684/86285004-e0012b80-bbb1-11ea-925b-0054c40f5f22.png)
![image](https://user-images.githubusercontent.com/37679684/86285128-13dc5100-bbb2-11ea-8024-daf2d9ae7b7f.png)

After:
![image](https://user-images.githubusercontent.com/37679684/86285083-0030ea80-bbb2-11ea-8f3b-303a57e8d4a2.png)
![image](https://user-images.githubusercontent.com/37679684/86285113-0aeb7f80-bbb2-11ea-9d0e-43196925759f.png)

cpldcpu commented:
Thank you!

Have you checked on desktop as well? Looks a bit weird to me:

![grafik](https://user-images.githubusercontent.com/4086406/86354295-1352c180-bc69-11ea-8629-0c9034a0d82b.png)

Kashouryo commented:
Yeah, I split the easy pdk programmer section to two divs. They will be horizontal on desktop but vertical on mobile. I removed the table's margin, now it should look a bit better. Or should I keep them vertical?
![image](https://user-images.githubusercontent.com/37679684/86412773-aef62900-bc8d-11ea-8536-dada1d743afc.png)

cpldcpu commented:
closing this due to roll out of the Jekyll based site. Thanks a lot anyways, and feel free to contribute to the other version!
